plugins/statistics.koplugin/main.lua

```diff
@@ -280,31 +422,95 @@ function ReaderStatistics:getTotalStats()
            total_stats
 end
 
+
+function ReaderStatistics:getAdvStats()
+    local total_stats = {
+        {
+            T(_"Last Week"),"",
+            callback = function()
+                UIManager:show(KeyValuePage:new{
+                    title = T(_"Week Statistics"),
```

Comment:
Since we are not doing templating for these string, you can just write it as `_("Weekly statistics")`. `T` function is not required.